### DISCUSSION TITLE
Freeplane title bar - Windows

### DISCUSSION THREAD

**Discussion by lplonka** [freeplane/freeplane#1716] [Help]
[Windows 10] Freeplane title bar stopped changing color when Freeplane window gets or loses focus. I need this while working so I can see which window has "focus" and where, for example, I paste something with ctrl+v. Is there any method to restore the "normal" title bar?

**Comment by dpolivaev** [comment]
Why has the title bar stopped changing colors? Was it after the update to a certain Freeplane version, after the Windows update, or after the Java update? Does the older Freeplane version work as before? Which Freeplane version has introduced this issue?

**Reply by lplonka** [reply]
The problem occurred in a way I don't understand. I was using version 1.11.10, and I use Freeplane practically every day. Yesterday, all of a sudden Freeplane lost all my settings and started up "cleanly" meaning, for example, it didn't load the maps I last had open. The startup of the program was accompanied by a beep, which I was not aware of before.
It turned out that it generally did not load my settings.
I mistakenly thought that maybe it had updated itself or something....
I changed the settings as I like. Today I open Freeplane and look and here the title bar is gone. I went through all the settings in the program and didn't notice an option specifically related to the title bar >>> so I wrote this post.
Then I noticed that there is a new version of the program - 1.11.11
After installing version 1.11.11, the program stopped displaying maps - it showed only one node from the entire map on a blue background. 
After many attempts, the description of which I will not litter the forum >> managed to run the program in such a way that I deleted the folder with user settings. 
Now suddenly everything is OK and the program displays maps and I can see the title bar.
Sorry for the long story I hope that the information that emergency deletion of settings sometimes helps - may be helpful.
